### PR TITLE
Rychkov Tymur +task2

### PR DESCRIPTION
**Pull request by tvrych**

### CODE REVIEW

**Review by Dmitriy-NK — CHANGES_REQUESTED** [review]
Удалите все добавленные файлы кроме Rychkov_Tasks/task1/Task1.ipynb  с использованием "git rm file_path".
Или создайте форк заново с изменениями только для файла Rychkov_Tasks/task1/Task1.ipynb

**Comment by tvrych** [comment]
вроде поправил

**Review by Dmitriy-NK — CHANGES_REQUESTED** [review]
Удалите все добавленные файлы кроме Rychkov_Tasks/task1/Task1.ipynb. Оставьте только файлы на момент создания форка (Удалите Tasks/task1/Titanic_template.ipynb, Practice/Fishers Iris/Iris.ipynb, Tasks/task2/FashionMnist_template.ipynb ) . В README верните строку # 2022-knu-yolo остальное удалите.

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
-# 2022-knu-yolo
\ No newline at end of file
+# 2022 KNU YOLO practicum page
```
Comment (Dmitriy-NK) [inline README.md:1]: В README верните строку `# 2022-knu-yolo`, остальное удалите.

**Review by Dmitriy-NK — CHANGES_REQUESTED** [review]
1. Для нескольких графиков подряд лучше использовать вариант предложенный в шаблоне задания с их комбинированием в одну панель, это значительно улучшает читаемость и дает возможность анализа сразу нескольких зависимостей.
2. График 4 (в этом наборе графиков) построен с нормированием к полному числу пассажиров. Предполагается сначала разбиение на подмножества а потом для каждого подмножества построение графика. у seaborn.kdeplot для этого используется аргумент "common_norm".
3. В разделе "graph down by gender" отсутствует график пропорционального распределения "# Next adjust graph to display the proportions of survival by gender" и просьба добавить несколько предложений по своей оценке после "Add your summary about recieved data for raw value counts and proportionally"
4. В следующем разделе задания Pclass structure была  просьба построить данные для 2 подмножеств (объединенный 1 и 2 класс) и 3 класс.  Также добавьте график с учетом значения Выживших и нет в данных подмножествах.
5. Внимательней проанализируйте данные тепловой карты для confusion matrix, они расходятся с датасетом. по умолчанию sklearn.metrics.confusion_matrix считает метку 0 - положительной (это связано с возможностью классификации с многими метками) так собственный порядок можно задать в аргументе labels, в нашем случае [1,0]  [detailes](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)
6. Для построения ROC Кривых использованы некорректные данные. Данный момент поднимался во время практикума в задании с ирисами. Для построения соответствующего графика нужны данные из predict_proba метода

**Comment by tvrych** [comment]
поправил все ошибки, кроме тепловой карты, по идее правильно

**Review by Dmitriy-NK — COMMENTED** [review]
3-й и 4-й пункты - все еще вопросы к отсутствию графиков. 5-й п. матрица несоответствий некорректна.

**Comment by tvrych** [comment]
доисправлял

**Review by Dmitriy-NK — COMMENTED** [review]
Этого графика [пример для 3](https://github.com/srk-practicum/2022-knu-yolo/issues/35) всё ещё нет. И просьба нормировать значения в матрице,  у sklearn.metrics.confusion_matrix сесть аргумент для такого  [detailes](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)

**Comment by tvrych** [comment]
Добавил второй таск, поправил первый

**Review by Dmitriy-NK — APPROVED** [review]
Task 2 approved
